src/node.js

```diff
@@ -22,6 +22,64 @@
 
     EventEmitter.call(process);
 
+    var cluster;
+    // Default process warning handler..
+    process.on('warning', (warning) => {
+      if (!cluster)
```

Comment:
Why add this complexity?  `process.on('error` doesn't behave like this in cluster mode, does it?